testnets/injectivetestnet/chain.json

```diff
@@ -33,33 +33,39 @@
   },
   "codebase": {
     "git_repo": "https://github.com/InjectiveLabs/testnet",
-    "recommended_version": "v1.10.2",
+    "recommended_version": "v1.11.1",
     "compatible_versions": [
-      "v1.10.2"
+      "v1.11.1"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/InjectiveLabs/testnet/releases/download/v1.10.2-1678712142/linux-amd64.zip",
-      "darwin/amd64": "https://github.com/InjectiveLabs/testnet/releases/download/v1.10.2-1678712142/darwin-amd64.zip"
+      "linux/amd64": "https://github.com/InjectiveLabs/testnet/releases/download/v1.11.1-1685205489/linux-amd64.zip",
+      "darwin/amd64": "https://github.com/InjectiveLabs/testnet/releases/download/v1.11.1-1685205489/darwin-amd64.zip"
     },
     "genesis": {
       "genesis_url": "aws s3 cp s3://injective-snapshots/testnet/genesis.json . --no-sign-request"
     },
     "versions": [
       {
-        "name": "v1.10.2",
```

Comment:
Instead of writing over the v1.10 version object, can we instead create a v1.11 object, please?